docs/README.md

```diff
@@ -2,32 +2,42 @@
 
 Built with [MkDocs](https://www.mkdocs.org/) using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
 
-## Developing
+## Developing Locally
 
-### Prerequisites
+### Install Base Conda Environment
 
-This repository uses [Poetry](https://python-poetry.org/) to manage dependencies
+```
+mamba env create -f docs/doc-env.yml  
+mamba activate icechunk-docs
+```
 
-1. Install dependencies using `poetry install`
+### Build Icechunk Python
 
-### Running
+```
+cd icechunk-python
+maturin build
+pip install "$(ls ../target/wheels/*.whl | head -n 1)[docs]"
```

Comment:
```suggestion
pip install "$(ls -t ../target/wheels/*.whl | head -n 1)[docs]"
```

to ensure using the most recent wheel locally. On cloud this isn't an issue because there will only ever be one wheel in that directory.